vgscollect/src/main/java/com/verygoodsecurity/vgscollect/view/card/filter/CardInputFilter.kt

```diff
@@ -4,4 +4,5 @@ package com.verygoodsecurity.vgscollect.view.card.filter
 interface CardInputFilter {
     fun clearFilters()
     fun addFilter(filter: VGSCardFilter?)
+    fun setAllowToOverrideDefaultValidation(canOverride: Boolean)
```

Comment:
@Endorf I think InputRunnable should not implement this interface as it leads to low cohesion of `InputRunnable`. I think `CardInputFilter` interface should be implemented only by `InputCardNumberConnection` as looks like only this class use this feature.